src/coreclr/src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComponentActivator.cs

```diff
@@ -21,11 +21,6 @@ public static class ComponentActivator
 
         private static string MarshalToString(IntPtr arg, string argName)
         {
-            if (arg == IntPtr.Zero)
-            {
-                throw new ArgumentNullException(argName);
-            }
-
 #if TARGET_WINDOWS
             string? result = Marshal.PtrToStringUni(arg);
 #else
```

Comment:
This can also just call `Marshal.PtrToStringAuto` and avoid the ifdef.

Reply:
Interesting, this isn't documented. I can do this. Where should I file an issue for the missing documentation?

Reply:
https://github.com/dotnet/docs/ is the place to open doc issues 

@jkoritzinsky https://github.com/dotnet/coreclr/pull/23664 mentioned that the documentation should have been updated. I guess it fell through the cracks.